### PR TITLE
Fix wrong text in assertTrue API

### LINKED ISSUE #2080: Wrong text for assertTrue
I still have the same problem with the fault description of assertEquals and assertTrue, that was supposed to be solved.

My pom.xml:

```xml
        <!-- https://mvnrepository.com/artifact/org.testng/testng -->
        <dependency>
            <groupId>org.testng</groupId>
            <artifactId>testng</artifactId>
            <version>7.0.0-beta3</version>
            <scope>test</scope>
        </dependency>
```

My test file:

```java
import org.testng.annotations.Test;

import static org.testng.Assert.assertTrue;

public class testTestNg {
    @Test
    public void showFail() {
        assertTrue(false, "MyMessage");
    }

    @Test(expectedExceptions = AssertionError.class, expectedExceptionsMessageRegExp = "MyMessage did not expect to find \\[true\\] but found \\[false\\]")
    public void version7Behavior() {
        assertTrue(false, "MyMessage");
    }

    @Test(expectedExceptions = AssertionError.class, expectedExceptionsMessageRegExp = "MyMessage expected \\[true\\] but found \\[false\\]")
    public void version6Behavior() {
        assertTrue(false, "MyMessage");
    }
}
```

I the test "showFail", I get:
`java.lang.AssertionError: MyMessage did not expect to find [true] but found [false]`

which is bad language. Isn't that exactly what this issue is about?

_Originally posted by @HelgeStenstrom in https://github.com/cbeust/testng/pull/1936#issuecomment-492800979_

### PR DESCRIPTION
**Pull request by jiong-fu**
Closes #2080

Fixes # .

### Did you remember to?

- [x] Add test case(s)

- [x] Update `CHANGES.txt`

We encourage pull requests that:

* Add new features to TestNG (or)
* Fix bugs in TestNG

If your pull request involves fixing SonarQube issues then we would suggest that you please discuss this with the 
[TestNG-dev](https://groups.google.com/forum/#!forum/testng-dev) before you spend time working on it.

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Current
+Fixed: GITHUB-2080: Wrong text for assertTrue
```
Comment (krmahadevan) [inline CHANGES.txt]: You might want to add your name along with the bug information.

Comment (jiong-fu) [reply CHANGES.txt]: Hi Krishnan, I have obviously ignored the names that other contributors wrote in the CHANGE.txt, thanks for reminding!
In addition, could you help to figure out why the Travis build failed? I found that it failed also for the two previous commits before mine...

**Comment by juherr** [comment]
@jiong-fu Looks like a gradle issue: https://scans.gradle.com/s/jdtmjd5exop2u/failure?openFailures=WzBd&openStackTraces=WzEsMF0#top=0

Maybe you can fix it by upgrading Gradle.

**Review by juherr — APPROVED** [review]

**Comment by krmahadevan** [comment]
@juherr - We cannot upgrade gradle. Looks like @cbeust downgraded gradle as part of [this](https://github.com/cbeust/testng/commit/2984d735e8c122e52cea31466d981f1d00835b09) commit to fix a signing issue while publishing. So if we upgrade, then we break that.

**Comment by juherr** [comment]
Sure but the issue could be fixed in a recent release. The current version is 5.6.

**Comment by krmahadevan** [comment]
@juherr - That is the part that I can't verify because its involving the signing step of the jar just before publishing. So I think we may have to wait for @cbeust to come back on this.

**Comment by jiong-fu** [comment]
@juherr +1 not sure if @cbeust has tried to upgrade Gradle to the latest version, if not let's give it a try. :-)

**Comment by jiong-fu** [comment]
@krmahadevan may I merge the pull request now? (although it is a little bit weird to close a pull request with failed CI build)

**Review by krmahadevan — APPROVED** [review]

**Pull request merged**